pkg/eventfilter/attributes/filter.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package attributes
+
+import (
+	"context"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"go.uber.org/zap"
+	"knative.dev/pkg/logging"
+
+	eventingv1beta1 "knative.dev/eventing/pkg/apis/eventing/v1beta1"
+	"knative.dev/eventing/pkg/eventfilter"
+)
+
+type AttributesFilter map[string]string
+
+func NewAttributesFilter(attrs map[string]string) eventfilter.Filter {
+	return AttributesFilter(attrs)
+}
+
+func (attrs AttributesFilter) Filter(ctx context.Context, event cloudevents.Event) eventfilter.FilterResult {
```

Comment:
Add comment

Reply:
I added the comment on the interface itself